frontend/src/shared/helpers/attribute.helpers.ts

```diff
@@ -0,0 +1,33 @@
+import { CrowdIntegrations } from '@/integrations/integrations-config';
+import { isEqual } from 'lodash';
+
+export const getAttributeSources = (attribute: Record<string, string>): string[] => {
+  const defaultValue: string | undefined = attribute.default;
+  return Object.keys(attribute).filter((key) => key !== 'default' && isEqual(attribute[key], defaultValue));
+};
+
+export const getAttributeSourceName = (attribute: Record<string, string>): string | null => {
+  if (!attribute) {
+    return null;
+  }
+  const sources = getAttributeSources(attribute);
+  if (sources.length === 0) {
+    return null;
+  }
+
+  // Sort that integrations are first, then others like enrichment and last is custom
+  const prioritySortedSources = sources.sort((a, b) => {
+    const aConfig = !!CrowdIntegrations.getConfig(a)?.name;
+    const bConfig = !!CrowdIntegrations.getConfig(b)?.name;
+
+    if (a === 'custom') return 1; // 'custom' should be last
```

Comment:
<img width="166" alt="Screenshot 2023-11-22 at 10 59 30" src="https://github.com/CrowdDotDev/crowd.dev/assets/20134207/6b6d4580-51ed-4eda-acba-c0652bd659e3">

We shouldn't show the source for Custom. Otherwise we would have to do it for everything that's editable